docsets/HAProxy/docset.json

```diff
@@ -1,12 +1,20 @@
 {
     "name": "HAProxy",
-    "version": "2.7.0",
+    "version": "2.9.0",
     "archive": "HAProxy.tgz",
     "author": {
         "name": "Ruoshan Huang",
```

Comment:
Would you like to update the author section as well? Though I don't know the right format for having multiple names. cc @Kapeli